snc.sh

```diff
@@ -107,6 +107,14 @@ if [ -f /etc/NetworkManager/dnsmasq.d/openshift.conf ]; then
     sudo rm /etc/NetworkManager/dnsmasq.d/openshift.conf
 fi
 
+destroy_libvirt_resource
+sed -e "s|NETWORK_NAME|${SNC_PRODUCT_NAME}|" \
+    -e "s|CLUSTER_NAME|${SNC_PRODUCT_NAME}|" \
+    -e "s|BASE_DOMAIN|${BASE_DOMAIN}|" \
+    net.xml.template > net.xml
```

Comment:
This could be moved to `create_libvirt_resource`?

Reply:
Sure we can move it to `create_libvirt_resource helper.